### PR TITLE
[GER] Translation of AImod and 'Adding custom hitsounds'

### PR DESCRIPTION
**Pull request by miterosan**

### CODE REVIEW

**Comment by TPGPL** [comment]
markdown seems good :+1:

**Comment by miterosan** [comment]
@TPGPL I used the md of the english version

**Comment by miterosan** [comment]
@Loctav are you sure about removing **du**?

It is incredibly hard at some places. :/

**Comment by miterosan** [comment]
@Loctav How about this version?

**Comment by miterosan** [comment]
Could somebody review this again?

File: wiki/Beatmap_Editor/AiMod/de.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+[AiMod](AiMod "wikilink") ist ein Programm, welchen man im Beatmap Editor finden kann.
+Der [AiMod](AiMod "wikilink") kann über den Tastenkürzel **\[Strg\] + \[Shift\] + \[A\]** oder über **File**, dann **Open AiMod** geöffnet werden.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/de.md]: same thing (remove link; change `[AiMod](AiMod "wikilink")` to `AiMod`)

---

for keyboard keys, do not surround the key with square brackets. Instead, use code (this: `` ` ``).
So it should be:
`Strg` + `Shift` + `A`
`` `Strg` + `Shift` + `A` ``

File: wiki/Beatmap_Editor/AiMod/de.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+[AiMod](AiMod "wikilink") ist ein Programm, welchen man im Beatmap Editor finden kann.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/de.md]: remove link; change `[AiMod](AiMod "wikilink")` to `AiMod`

File: wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+===========================
+
+Von dem Thread: [*Wie man einen Eigenen Hitsound hinzufügt* von: neonat](https://osu.ppy.sh/forum/p/3215699)
+
+Eigene Hitsounds
+---------------------------
+
+Als erstes werden einige Hitsounds benötigt.
+
+-   Man kann sie aus der [Custom Hitsound Library](Custom_Hitsound_Library "wikilink") bekommen.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md]: change the link to `[Custom Hitsound Library](/wiki/Beatmap_Editor_Guides/Custom_Hitsound_Library/)`
If `Custom Hitsound Library` is translatable, go ahead and translate it.

**Comment by miterosan** [comment]
Megaapplepi,  could you confirm?

File: wiki/Beatmap_Editor/AiMod/de.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+**AiMod** ist ein Programm, welchen man im Beatmap Editor finden kann.
+Der **AiMod** kann über den Tastenkürzel **\[Strg\] + \[Shift\] + \[A\]** oder über **File**, dann **Open AiMod** geöffnet werden.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/de.md]: change `**\[Strg\] + \[Shift\] + \[A\]**` to `` `Strg` + `Shift` + `A` ``

File: wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+===========================
+
+Von dem Thread: [*Wie man einen Eigenen Hitsound hinzufügt.* von: neonat](https://osu.ppy.sh/forum/p/3215699)
+
+Eigene Hitsounds
+---------------------------
+
+Als erstes werden einige Hitsounds benötigt.
+
+-   Man kann sie aus der [Spieler-Hitsound-Bibliothek](Custom_Hitsound_Library "wikilink") bekommen.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Adding_Custom_Hitsounds/de.md]: change `[Spieler-Hitsound-Bibliothek](Custom_Hitsound_Library "wikilink")` to ` [Spieler-Hitsound-Bibliothek](../Custom_Hitsound_Library)`

**Comment by miterosan** [comment]
@MegaApplePi  done

File: wiki/Beatmap_Editor/AiMod/de.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+**AiMod** ist ein Programm, welchen man im Beatmap Editor finden kann.
+Der **AiMod** kann über den Tastenkürzel **Strg + Shift + A** oder über **File**, dann **Open AiMod** geöffnet werden.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/AiMod/de.md]: change `**Strg + Shift + A**` to `` `Strg` + `Shift` + `A` ``
... I've asked this specific change 3 times already, is there a reason to not do this?

Comment (miterosan) [inline wiki/Beatmap_Editor/AiMod/de.md]: I'm sorry .-. I didn't see that thin lines .-.

**Comment by Loctav** [comment]
Still not 100% happy, but I think it's a good starting point to work with. Can always be improved from there on, at least it isnt full of grammar issues anymore

**Pull request merged**